src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -97,55 +97,76 @@ This section lists required services and some required system configuration.
+The following table summarizes the recommendations of the HBase community with respect to deploying
+on various Java versions. The icon:check-circle[role="green"] symbol indicates a base level of
 testing and willingness to help diagnose and address issues you might run into. Similarly, an entry
-of icon:exclamation-circle[role="yellow"] or icon:times-circle[role="red"] generally means that
-should you run into an issue the community is likely to ask you to change the Java environment
-before proceeding to help. In some cases, specific guidance on limitations (e.g. whether compiling
-/ unit tests work, specific operational issues, etc) will also be noted.
+of icon:exclamation-circle[role="yellow"] means that, should you run into an issue, the community
+is likely to ask you to change the Java environment rather than proceeding to help on that Java
+version. This applies categorically to Non-LTS releases (see below). The
+icon:times-circle[role="red"] means that support has been dropped for this version or has been
+attempted and is known to not work. For combinations of newer JDK with older HBase releases, it's
+likely that there are known compatibility issues that cannot be addressed under our compatibility
+guarantees, making that combination impossible. In some cases, specific guidance on limitations
+(e.g. whether compiling / unit tests work, specific operational issues, etc) will also be noted.
+
+.Long-Term Support JDKs are Recommended
+[WARNING]
+====
+HBase recommends downstream users rely on JDK releases that are marked as Long-Term Supported
+(LTS), either from the OpenJDK project or vendors. At the time of this writing, the following JDK
+releases are NOT LTS releases and are NOT supported by the Apache HBase community: JDK9, JDK10,
+JDK12, JDK13, and JDK14. Community discussion around this decision is recorded on
+link:https://issues.apache.org/jira/browse/HBASE-20264[HBASE-20264].
+====
 
-.Long Term Support JDKs are recommended
+.HotSpot vs. OpenJ9
 [TIP]
 ====
-HBase recommends downstream users rely on JDK releases that are marked as Long Term Supported (LTS)
-either from the OpenJDK project or vendors. As of March 2018 that means Java 8 is the only
-applicable version and that the next likely version to see testing will be Java 11 near Q3 2018.
+At this time, all testing performed by the Apache HBase project runs on the HotSpot variant of the
+JVM. When selecting your JDK distribution, please take this into consideration.
 ====
 
 .Java support by release line
-[cols="6*^.^", options="header"]
+[cols="4*^.^", options="header"]
 |===
-|HBase Version
-|JDK 7
-|JDK 8
-|JDK 9 (Non-LTS)
-|JDK 10 (Non-LTS)
-|JDK 11
-
-|2.1+
+|Java Version
+|HBase 1.3+
+|HBase 2.1+
+|HBase 2.3+
+
+|JDK6
+|icon:times-circle[role="red"]
 |icon:times-circle[role="red"]
+|icon:times-circle[role="red"]
+
+|JDK7
+|icon:check-circle[role="green"]
+|icon:times-circle[role="red"]
+|icon:times-circle[role="red"]
+
+|JDK8
 |icon:check-circle[role="green"]
```

Comment:
JDK8 should be `red` for HBase 1.3+ right? branch-1.3 and 1.4 have `maven-compiler-plugin` source and target as 1.7.

Reply:
Although this doesn't mean that users can't compile 1.3+ with JDK 1.8, but branch-1 is still not 1.8 compatible source code wise. I am not sure whether this means we should mark this `red` because ppl can run clusters on Java 8 env.

Reply:
We're talking about runtime support here. You're right though, the language is not clear. Let me take another swing.

Reply:
yeah this chart has always been about runtime version supports. good clarification.